CHANGELOG.md

```diff
@@ -124,7 +124,7 @@
 * Added methods for convenience checking Navigation tiles in a `TileStore`: `TileStore.containsLatestNavigationTiles(forCacheLocation:completion:)`, `TileStore.tileRegionContainsLatestNavigationTiles(forId:cacheLocation:completion:)` and methods for getting Navigation tiles from `TilesetDescriptorFactory`: `TilesetDescriptorFactory.getLatest(forCacheLocation:completionQueue:completion)`, `TilesetDescriptorFactory.getSpecificVersion(forCacheLocation:version:completionQueue:completion:)`. ([#3015](https://github.com/mapbox/mapbox-navigation-ios/pull/3015), [#3164](https://github.com/mapbox/mapbox-navigation-ios/pull/3164))
 * Fixed a thread-safety issue in `UnimplementedLogging` protocol implementation. ([#3024](https://github.com/mapbox/mapbox-navigation-ios/pull/3024))
 * Implemented automatic switching to older navigation tiles versions (if they are present in current `TileStore`) and back. You can subscribe to `Notification.Name.navigationDidSwitchToFallbackVersion` and `Notification.Name.navigationDidSwitchToTargetVersion` notifications to monitor such events. ([#3014](https://github.com/mapbox/mapbox-navigation-ios/pull/3014))
-* Fixed an issue where the current road name label sometimes displayed the name of an intersecting road instead of the current road or blinked in and out. ([#3019](https://github.com/mapbox/mapbox-navigation-ios/pull/3019))
+* Fixed an issue where the current road name label sometimes displayed the name of an intersecting road instead of the current road or blinked in and out with `NavigationStatus.roadName`. ([#3257](https://github.com/mapbox/mapbox-navigation-ios/pull/3257))
```

Comment:
`NavigationStatus.roadName` is an internal implementation detail, so I removed the reference to it in 416744d746381874c63eadde3ef114ee8c6f4296.